### PR TITLE
set up deployment

### PR DESCRIPTION
**Pull request by koggs**
### What I did:
-Implement Heroku
-Remove jquery from application.js
-Create event controller

### CODE REVIEW

File: app/assets/javascripts/eventse.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```
Comment (juliaramstedt) [inline app/assets/javascripts/eventse.js]: This file and `app/assets/stylesheets/eventse.scss` was probably autogenerated with `rails g controller`, if you don't think that you'll be using them soon, you can remove them

File: app/helpers/eventse_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module EventseHelper
+end
```
Comment (juliaramstedt) [inline app/helpers/eventse_helper.rb]: You can remove this file as well, or rename it if you think you'll use it

**Comment by juliaramstedt** [comment]
The PR doesn't actually set up Heroku :) but cleanup the auto-generated files that you won't need, and it looks good!

File: app/assets/javascripts/eventse.js (hunk 1 above)
Comment (koggs) [reply app/assets/javascripts/eventse.js]: done

File: app/helpers/eventse_helper.rb (hunk 2 above)
Comment (koggs) [reply app/helpers/eventse_helper.rb]: done

File: spec/controllers/eventse_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe EventseController, type: :controller do
```
Comment (juliaramstedt) [inline spec/controllers/eventse_controller_spec.rb]: Remove ’e’ (in file name as well)

File: app/assets/stylesheets/eventse.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Eventse controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (juliaramstedt) [inline app/assets/stylesheets/eventse.scss]: Remove ”e” in file name

File: spec/helpers/eventse_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+# the EventseHelper. For example:
+#
+# describe EventseHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe EventseHelper, type: :helper do
```
Comment (juliaramstedt) [inline spec/helpers/eventse_helper_spec.rb]: Remove extra ”e” at the end, here and in the file name

Comment (koggs) [reply spec/helpers/eventse_helper_spec.rb]: 👍🏻

File: app/assets/stylesheets/eventse.scss (hunk 4 above)
Comment (koggs) [reply app/assets/stylesheets/eventse.scss]: 👍🏻

**Pull request merged**